### PR TITLE
[gsoc-ideas-2020] Grimoirelab

### PR DESCRIPTION
**Pull request by valeriocos**
This commit proposes 3 ideas that target the grimoirelab platform.

The micro-tasks will be available soon.

### CODE REVIEW

File: GSoC-Ideas.md [hunk 1]
```diff
@@ -1,3 +1,67 @@
 # Ideas for Google Summer of Code projects
 
-## Idea: TBD
+## Idea: Enhancing ELK functionalities
+
+[ Micro-tasks and place for questions ](https://github.com/chaoss/grimoirelab/issues/...)
```
Comment (GeorgLink) [inline GSoC-Ideas.md]: issue link incomplete in line 5

File: GSoC-Ideas.md [hunk 2]
```diff
@@ -1,3 +1,67 @@
...
+
+* _Difficulty:_ Medium
+* _Requirements:_ Python programming. Interest in software analytics. Willingness to understand GrimoireLab internals.
+* _Recommended:_ Experience with ElasticSearch and Pandas would be convenient, but can be learned during the project.
+* _Mentors:_ @sduenas, @valeriocos, @zhquan
+
+
+## Idea: Enabling the reuse of CHAOSS metrics using Kibana-based dashboards
+
+[ Micro-tasks and place for questions ](https://github.com/chaoss/grimoirelab/issues/...)
```
Comment (GeorgLink) [inline GSoC-Ideas.md]: issue link incomplete in line 28

**Comment by valeriocos** [comment]
Sure, I'm preparing them. I'll ping you later, thanks

File: GSoC-Ideas.md [hunk 3]
```diff
@@ -1,3 +1,67 @@
...
+Other aims, such as enhancing Kidash or other components to support the implementation of the approach are completely within scope.
+
+* _Difficulty:_ Medium
+* _Requirements:_ Python programming. Interest in software analytics. Willingness to understand GrimoireLab internals.
+* _Recommended:_ Experience with ElasticSearch and Kibana would be convenient, but can be learned during the project.
+* _Mentors:_ @alpgarcia, @sduenas, @valeriocos
+
+## Idea: Support for Quality Models based on CHAOSS metrics
+
+[ Micro-tasks and place for questions ](https://github.com/chaoss/grimoirelab/issues/...)
```
Comment (GeorgLink) [inline GSoC-Ideas.md]: issue link incomplete in line 49

**Comment by GeorgLink** [comment]
Thanks @valeriocos

**Comment by valeriocos** [comment]
@GeorgLink ready for review

**Pull request merged**